test/TestAppUtils/Themes/Generic.xaml

```diff
@@ -39,7 +39,8 @@
                                 <StackPanel Orientation="Horizontal">
                                     <Button x:Name="BackButton" Margin="0,0,4,0"
                                         Style="{ThemeResource AccentButtonStyle}"
-                                        AutomationProperties.AutomationId="__BackButton" 
+                                        AutomationProperties.AutomationId="__BackButton"
+                                        AutomationProperties.Name="Back"
```

Comment:
Needed to add that since otherwise, Axe would always return at least one a11y issue.